.gitlab/validate-agent-build/trigger_agent_build.py

```diff
@@ -0,0 +1,43 @@
+import time
+
+import requests
+import os
+import argparse
+
+
+DATADOG_AGENT_PIPELINE_ID = 22
```

Comment:
I would get this from an environment variable, just to leak less information to the public.